provider_settings.json

```diff
@@ -1,12 +1,12 @@
 [
```

Comment:
@shawn-hurley can we make this file a sample and add the actual file to gitignore so you don't have to check in your local paths in repo?

Reply:
Fixed in the other PR.